next-env.d.ts

```diff
@@ -1,3 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
... This link explains nothing about why this file can't be edited and thus this comments adds no real information. no need for the comment to exist

Reply:
when I run the project, this line is always added... I will remove